test/rfqm_test.ts

```diff
@@ -99,12 +103,43 @@ describe(SUITE_NAME, () => {
         const quoteRequestor = new QuoteRequestor({}, mockAssetOfferings, axiosClient);
 
         // Create the mock rfqBlockchainUtils
+        const validationResponse: [BigNumber, BigNumber] = [new BigNumber(1), new BigNumber(1)];
+        const validateMetaTransactionPromise = new Promise<[BigNumber, BigNumber]>((resolve) => {
+            resolve(validationResponse);
+        });
         const rfqBlockchainUtilsMock = mock(RfqBlockchainUtils);
         when(
             rfqBlockchainUtilsMock.generateMetaTransaction(anything(), anything(), anything(), anything(), anything()),
         ).thenReturn(MOCK_META_TX);
+        when(
+            rfqBlockchainUtilsMock.generateMetaTransactionCallData(anything(), anything()),
+        ).thenReturn(MOCK_META_TX_CALL_DATA);
+        when(
+            rfqBlockchainUtilsMock.validateMetaTransactionOrThrowAsync(anything(), anything(), anything(), anything()),
+        ).thenReturn(validateMetaTransactionPromise);
         const rfqBlockchainUtils = instance(rfqBlockchainUtilsMock);
 
+        interface SqsResponse {
+            Id: string;
+            MD5OfMessageBody: string;
+            MessageId: string;
+        }
+        const sqsResponse: SqsResponse[] = [{
+            Id: 'id',
+            MD5OfMessageBody: 'MD5OfMessageBody',
+            MessageId: 'MessageId',
+        }];
+        const sqsEntryPromise = new Promise<SqsResponse[]>((resolve) => {
+            resolve(sqsResponse);
+        });
```

Comment:
can remove these lines if using the syntax below. But if not using syntax below, can express this succinctly as:
```suggestion
        const sqsEntryPromise = Promise.resolve(sqsResponse);
```